src/gui/mzroll/forms/settingsform.ui

```diff
@@ -1404,6 +1404,13 @@ p, li { white-space: pre-wrap; }
      </widget>
     </widget>
    </item>
+   <item row="1" column="0">
+    <widget class="QPushButton" name="resetButton">
+     <property name="text">
+      <string>Reset defaults</string>
```

Comment:
@rish9511 I'd suggest changing this to "Use Default Parameters". Much clearer for users

Reply:
@shubhra-agrawal  okay